.rubocop.yml

```diff
@@ -3,3 +3,6 @@ Metrics/LineLength:
 
 Metrics/BlockLength:
   Max: 100
+
+Style/SymbolArray:
+  Enabled: false
```

Comment:
@ahenroid Hi Andy I was going to do this in a separate PR after this one but more than happy to do in this one. Ill make the changes to the rakefile now. Cheers Mark.